package.js

```diff
@@ -30,8 +30,7 @@ Package.on_use(function (api) {
     'coffeescript@1.0.4',
     'underscore',
     'velocity:core@0.4.5',
-    'velocity:shim@0.1.0',
-    'velocity:test-proxy@0.0.4'
```

Comment:
I have removed this dependency because it causes a circular dependency. I think it is unnecessary because the node-soft-proxy does not use the velocity:test-proxy package.